style.css

```diff
@@ -0,0 +1,179 @@
+    background-position: center;
+    background-size: cover; 
+    grid-column-start: 1;
+    grid-column-end: 5;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    height:690px;
+}
+.medium-fig-2 {
+    background-image: url("images/fig-two.jpg");
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-3 {
+    background-image: url("images/fig-three.jpg");
+    grid-column-start: 2;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-4 {
+    background-image: url("images/fig-four.jpg");
+    grid-column-start: 3;
+    grid-column-end: 4;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+.medium-fig-5 {
+    background-image: url("images/fig-five.jpg");
+    grid-column-start: 4;
+    grid-column-end: 5;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 200px;
+    width:100%;
+}
+/*nav bar: */
+.nav-bar-apple {    
+    left: 10%;  
+    margin-top: 5%;
+    position: absolute;
+    width: 80%;
+}
+.nav-bar-apple li {
+    
+    border-right: 1px solid grey;
+    display: table-cell;
+    height: 36px;
+    overflow: hidden;      
+    text-align: center;
+    vertical-align: middle;
+    width: 7%;    
+}
+.nav-bar-apple ul {
+    list-style-type: none;
+    background-image: linear-gradient(to bottom, #8A8A8A 20%, #363737 80%);
+}
+.nav-bar-apple li:hover {
+   background-image: linear-gradient(to bottom,  #363737 20%, #8A8A8A 80%);
+}
+.nav-bar-apple ul li a { 
+    color: white;
+    font-size: 12px;
+    font-weight: bold; 
+    margin: 15px;
+    text-decoration: none;        
+}
+.nav-bar-apple li a img {    
+    position: relative;
+    top:5px;
+}
+#search {
+    position: relative;
+    top: 2%;
+}
+input {
+    background-color: #6e6e6e;
+    background-image:url("images/mag-white.png");
+    background-repeat:no-repeat; 
+    border-radius: 25px;
+    position: relative;
+    top: -15px;
+    transition: width 1s;
+    width: 50%;
+    -webkit-transition: width 1s;
+}
+input:focus { /* to change the color inside the input*/
+    background-color: white;
+    background-image: url("images/mag-grey.png");
+    background-repeat: no-repeat; 
+    width:70%;
+}
+/* pictures used as a phrases over the first image: */
+.first-phrase {
+    background-image: url("images/first-phrase.png");
+    background-repeat: no-repeat;
+    height: 100px;
+    left: 25%;
+    position: absolute;
+    top: 180px;
+    width: 100%;
+}
+.second-phrase {
+    background-image: url("images/second-phrase.png");
+    background-repeat: no-repeat;
+    height:100px;
+    left:22%;
+    position: absolute;
+    top: 275px;
+    width:100%;
+}
+.last-phrase {
+    background-image: url("images/last-phrase.png");
+    background-repeat: no-repeat;
+    height:100px;
+    left: 30%;
+    position: absolute;
+    top: 400px;   
+    width:100%;
```

Comment:
`SUGGESTION`

Consider changing the `width` length value to `70%;` **(width - left)**. Same issue as in [this comment]()